service/scheduler.go

```diff
@@ -105,7 +122,7 @@ func (s *Scheduler) cancelWorkflows(
 ) map[string]string {
 	updatedStatuses := statuses
 	for orchardID, status := range statuses {
-		if status == Activated.ToString() {
+		if status == Activated.ToString() || status == Created.ToString() {
```

Comment:
`delete` works better if the workflow is created but not activated. When it failed to create, we prefer `delete` the previously created ones, and when it failed to activate, we can only cancel.

Reply:
and this can just take a workflow ID and send the call, probably no need to check statuses anymore

Reply:
reverted back the original Schedule.cancelWorkflows() logic.  

activate if get orchard error, now calls orchard to delete workflow.  

I thought about leaving the status as Created, so the activate next tick would retry.  But it would retry every second without any terminal condition.  Thus activate if orchard error has to update status, I now put in attempt to delete workflow via orchard.  The sprinkler status would become `deleted` or `delete_failed`